README.md

```diff
@@ -48,24 +48,48 @@ Streamlit comes in with [a ton of additional powerful elements](https://docs.str
 
 
 <table border="0">
-   <tr>
-     <td><img src="https://user-images.githubusercontent.com/7164864/217936099-12c16f8c-7fe4-44b1-889a-1ac9ee6a1b44.png" style="max-height:150; width:auto; display:block;"></td>
-     <td><img src="https://user-images.githubusercontent.com/7164864/215110064-5eb4e294-8f30-4933-9563-0275230e52b5.gif"  style="max-height:150; width:auto; display:block;"></td>
-     <td><img src="https://user-images.githubusercontent.com/7164864/215174472-bca8a0d7-cf4b-4268-9c3b-8c03dad50bcd.gif" style="max-height:150; width:auto; display:block;"></td>
-     <td><img src="https://user-images.githubusercontent.com/7164864/217936149-a35c35be-0d96-4c63-8c6a-1c4b52aa8f60.png" style="max-height:150; width:auto; display:block;"></td>
-     <td><img src="https://user-images.githubusercontent.com/7164864/215173883-eae0de69-7c1d-4d78-97d0-3bc1ab865e5b.gif" style="max-height:150; width:auto; display:block;"></td>
-     <td><img src="https://user-images.githubusercontent.com/7164864/215109229-6ae9111f-e5c1-4f0b-b3a2-87a79268ccc9.gif" style="max-height:150; width:auto; display:block;"></td>
-
+  <tr>
+    <td>
+      <a href="https://docs.streamlit.io/library/api-reference/widgets">
+        <img src="https://user-images.githubusercontent.com/7164864/217936099-12c16f8c-7fe4-44b1-889a-1ac9ee6a1b44.png" style="max-height:150px; width:auto; display:block;">
+      </a>
+    </td>
+    <td>
+      <a href="https://docs.streamlit.io/library/api-reference/data/st.dataframe">
+        <img src="https://user-images.githubusercontent.com/7164864/215110064-5eb4e294-8f30-4933-9563-0275230e52b5.gif" style="max-height:150px; width:auto; display:block;">
+      </a>
+    </td>
+    <td>
+      <a href="https://docs.streamlit.io/library/api-reference/charts">
+        <img src="https://user-images.githubusercontent.com/7164864/215174472-bca8a0d7-cf4b-4268-9c3b-8c03dad50bcd.gif" style="max-height:150px; width:auto; display:block;">
+      </a>
+    </td>
+    <td>
+      <a href="https://docs.streamlit.io/library/api-reference/layout">
+        <img src="https://user-images.githubusercontent.com/7164864/217936149-a35c35be-0d96-4c63-8c6a-1c4b52aa8f60.png" style="max-height:150px; width:auto; display:block;">
+      </a>
+    </td>
+    <td>
+      <a href="https://docs.streamlit.io/library/get-started/multipage-apps/create-a-multipage-app">
```

Comment:
Think we may want the main MPA doc link here 
`https://docs.streamlit.io/library/get-started/multipage-apps`

Reply:
Sure! I'll just update the link.

Reply:
Done, lmk any other changes required.